.eslintrc.json

```diff
@@ -63,12 +63,12 @@
     "tsdoc/syntax": "error",
 
     // Typescript Rules
-    "@typescript-eslint/ban-ts-comment": "off",
-    "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-inferrable-types": "off",
-    "@typescript-eslint/no-non-null-asserted-optional-chain": "off",
-    "@typescript-eslint/no-non-null-assertion": "off",
-    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/ban-ts-comment": "warn",
+    "@typescript-eslint/no-explicit-any": "warn",
```

Comment:
Will changing these to `error` ensure that the changes you made don't get reinserted by making the workflow fail? If so, then change these to `error`

Reply:
Do I need to change the "warn" to "error"?

Reply:
I think so. When the inconsistencies are found the workflow should fail. We don't want a warning and no incentive for the submitter to fix the problem.

Reply:
So currently, I am unable to commit any more changes as the warnings are converted to errors and they are aborting the commit message.

Reply:
@parthiv360 if that is the case, first try to resolve those errors

Reply:
There are around 886 errors that are shown currently.